util/masking.py

```diff
@@ -184,7 +188,13 @@ def generate_ice_ring_resolution_ranges(beam, panel, params):
 @lru_equality_cache(maxsize=3)
 def _get_resolution_masker(beam, panel):
     logger.debug("resolution masker cache miss")
-    return ResolutionMaskGenerator(beam, panel)
+    t0 = time.perf_counter()
```

Comment:
Do we want to leave the timing in?

Reply:
I think it is possibly useful information to include (and is only debug log level), however I have removed the "resolution masker cache miss" logging statement above, so the net number of log statements is the same as before.